build.gradle

```diff
@@ -95,26 +95,26 @@ dependencies {
 	implementation ('net.fabricmc:mercury:0.3.0')
 
 	// Kotlin
-	implementation('org.jetbrains.kotlinx:kotlinx-metadata-jvm:0.5.0') {
+	implementation('org.jetbrains.kotlinx:kotlinx-metadata-jvm:0.6.0') {
```

Comment:
Was this tested, I have just updated to 0.6.0 and it seems to have broken and deprecated the old API we were using?

Reply:
We can revert it but It shouldn't have broken anything